Model/Sales/Repository/PackageRepository.php

```diff
@@ -93,48 +94,80 @@ public function fitInDigital_stamp()
             return false;
         }
 
-        if ($this->getWeight() > self::DEFAULT_WEIGHT) {
+        if ($this->isAllProductsFitInDigitalStamp() === false) {
```

Comment:
```suggestion
        if ($this->fitInDigitalStamp() === false) {
```

Reply:
Ben ik niet met je eens, in de functie `setFitInDigitalStampFromQuoteProducts` zit er namelijk een loop die kijkt of alle producten in een digitale postzegel kunnen. De uitkomt zet hij in `private $all_products_fit_in_digital_stamp`. Daarnaast bestaat de functie `fitInDigitalStamp` al.

Reply:
Done!